docs-devsite/ai.livegenerationconfig.md

```diff
@@ -0,0 +1,153 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# LiveGenerationConfig interface
+> This API is provided as a preview for developers and may change based on feedback that we receive. Do not use this API in a production environment.
+> 
+
+Configuration parameters used by [LiveGenerativeModel](./ai.livegenerativemodel.md#livegenerativemodel_class) to control live content generation.
+
+<b>Signature:</b>
+
+```typescript
+export interface LiveGenerationConfig 
+```
+
+## Properties
+
+|  Property | Type | Description |
+|  --- | --- | --- |
+|  [candidateCount](./ai.livegenerationconfig.md#livegenerationconfigcandidatecount) | number | <b><i>(Public Preview)</i></b> The maximum number of generated response messages to return. This value must be between 1 and 8. If unset, this will default to 1. |
```

Comment:
Is setting candidate count actually supported?  We don't document it right now and IIRC it's because it can't be changed?